src/test/java/com/artipie/asto/s3/S3StorageTest.java

```diff
@@ -105,7 +105,7 @@ void shouldUploadObjectWhenSaveContentOfUnknownSize(final AmazonS3 client) throw
         final String key = "unknown/size";
         this.storage().save(
             new Key.From(key),
-            new Content.From(new OneOffPublisher(ByteBuffer.wrap(data)))
+            new Content.OneTime(new Content.From(new OneOffPublisher(ByteBuffer.wrap(data))))
```

Comment:
@Sammers21 `OneOffPublisher` usage becomes pointless here and in all other places, this class may be safely removed